### PR TITLE
Added Bender.yml file

### PR DESCRIPTION
**Pull request by micprog**

### CODE REVIEW

**Comment by andreaskurth** [comment]
PR converted to Draft since the path for `riscv` is relative. Please change back once that is fixed.

File: Bender.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+package: 
+  name: fpu_interco
+  authors:
```
Comment (stmach) [inline Bender.yml:3]:
```suggestion
  authors:
    - "Igor Loi <igor.loi@greenwaves-technologies.com>"
```

File: Bender.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  name: fpu_interco
+  authors: 
+    - "Stefan Mach <smach@iis.ee.ethz.ch>"
+    - "Florent Rotenberg <florent.rotenberg@greenwaves-technologies.com>"
+    - "Andreas Kurth <akurth@iis.ee.ethz.ch>"
+    - "Francesco Conti <f.conti@unibo.it>"
+
+dependencies:
+  fpnew: { git: "git@github.com:pulp-platform/fpnew.git", version: 0.6.3 }
+  riscv: {path: "../riscv"} # Reference to be updated to git
```
Comment (stmach) [inline Bender.yml]:
```suggestion
  riscv: { git: "git@github.com:pulp-platform/riscv.git", version: master } # There don't seem to be recent releases
```

**Review by stmach — CHANGES_REQUESTED** [review]
Added suggestions for this

File: Bender.yml (hunk 2 above)
Comment (micprog) [reply Bender.yml]: Setting riscv to master will make this incompatible with other dependencies of pulp, as the up-to-date master refers to cv32e40p. There are some changes required to the [Bender.yml file in riscv](https://github.com/micprog/cv32e40p/commit/29a838f6e2811b12220cb13bb02b37bd8ccd012f), compared to the state included in pulp (tag pulpissimo-v3.4.0). I believe there is work being done to convert to cv32e40p, I haven't looked into it yet. I will temporarily set the dependency to a forked repo, I'm open for ideas on proper integration.

**Comment by micprog** [comment]
I updated the riscv dependency to a github repository, currently a fork as the cv32e40p core is not yet compatible with the rest of pulp. I believe work is being done in this repo on the cv32e40p_integration branch. I hope this is ok as a temporary solution. 
I should mention this is a not a push onto master, but the branch currently included in pulp_cluster. What is the status of merging this to master?

File: Bender.yml [hunk 3]
```diff
@@ -0,0 +1,31 @@
+package: 
+  name: fpu_interco
+  authors: 
+    - "Igor Loi <igor.loi@greenwaves-technologies.com>"
+    - "Stefan Mach <smach@iis.ee.ethz.ch>"
+    - "Florent Rotenberg <florent.rotenberg@greenwaves-technologies.com>"
+    - "Andreas Kurth <akurth@iis.ee.ethz.ch>"
```
Comment (andreaskurth) [inline Bender.yml]:
```suggestion
```
That's undeserved glory ;-) I did not contribute hardware code to this repository.

Comment (micprog) [reply Bender.yml]: I went by contributors to the github repository, but will update

**Pull request merged**